go/ct/rlz/conditions.go

```diff
@@ -724,11 +724,26 @@ func (c *isAddressCold) Restrict(generator *gen.StateGenerator) {
 }
 
 func (c *isAddressCold) GetTestValues() []TestValue {
-	return IsAddressWarm(c.key).GetTestValues()
+	property := Property(c.String())
+	return []TestValue{
+		NewTestValue(property, boolDomain{}, false, restrictAccountWarmCold(c.key)),
+	}
 }
 
 func (c *isAddressCold) String() string {
-	return fmt.Sprintf("cold(%v)", c.key)
+	return fmt.Sprintf("account warm(%v)", c.key)
```

Comment:
This makes `isAddressCold(param(0))` and `isAddressWarm(param(0))` indistinguishable when being listed e.g. as part of the `generator-info` command.